app/models/manageiq/providers/vmware/infra_manager/inventory/parser/host_system.rb

```diff
@@ -13,10 +13,8 @@ def network_ref_by_datacenter_and_name
     end
 
     def validate_host_system_props(object, props)
-      if props.fetch_path(:config).nil? || props.fetch_path(:summary).nil? || props.fetch_path(:summary, :config, :product).nil?
+      if props.fetch_path(:summary).nil? || props.fetch_path(:summary, :config, :product).nil?
```

Comment:
Should this be shortened to just `props.fetch_path(:summary, :config, :product).nil?`, since `props.fetch_path(:summary).nil?` is included inside that?  Not sure we want to do that if we want to clearly show we need both summary and product.

Reply:
Yeah I think we had it this way so that it was obvious what we were checking for, but I can address that in a comment rather than in code